project/src/components/const.ts

```diff
@@ -0,0 +1,13 @@
+
+export const enum AppRoutes {
```

Comment:
```suggestion
export const enum AppRoutes {
```
ну нужно делать пустой строки. можешь сделать константы, а в конце сделаешь экспорт.